ml_tools/eolearn/ml_tools/sampling.py

```diff
@@ -147,12 +147,15 @@ def get_mask_of_samples(image_shape: Tuple[int, int], row_grid: np.ndarray, colu
 class BaseSamplingTask(EOTask, metaclass=ABCMeta):  # noqa: B024
     """A base class for sampling tasks"""
 
-    def __init__(self, features_to_sample, *, mask_of_samples=None):
+    def __init__(
+        self,
+        features_to_sample: FeaturesSpecification,
+        *,
+        mask_of_samples: Optional[Tuple[FeatureType, Optional[str]]] = None,
```

Comment:
isn't this `Optional[SingleFeatureSpec]`?

Reply:
my brain say yes but mypy:
```bash
ml_tools/sampling.py:167:36: error: Incompatible types in assignment (expression has type "Tuple[FeatureType, Optional[str]]", variable has type "Union[Union[Tuple[Literal[FeatureType.BBOX, FeatureType.TIMESTAMP], None], Tuple[FeatureType, str]], Union[Tuple[Literal[FeatureType.BBOX, FeatureType.TIMESTAMP], None, None], Tuple[FeatureType, str, str]], None]")  [assignment]
ml_tools/sampling.py:185:21: error: Invalid index type "Union[Tuple[Literal[FeatureType.BBOX, FeatureType.TIMESTAMP], None], Tuple[FeatureType, str], Tuple[Literal[FeatureType.BBOX, FeatureType.TIMESTAMP], None, None], Tuple[FeatureType, str, str]]" for "EOPatch"; expected type "Union[FeatureType, Tuple[FeatureType, Union[str, None, ellipsis]]]"  [index]
```

Reply:
actually, this should be just `Optional[Tuple[FeatureType, str]]` because the `mask_of_samples` cannot be BBOX or TIMESTAMP

Reply:
done